newdle/client/src/components/answer/AnswerPage.js

```diff
@@ -26,6 +28,8 @@ export default function AnswerPage() {
   const availabilityData = useSelector(getAnswers);
   const allAvailableSelected = useSelector(isAllAvailableSelected);
   const allAvailableDisabled = useSelector(isAllAvailableSelectedImplicitly);
+  const dates = useSelector(getCalendarDates);
```

Comment:
This contains duplicates if a day has multiple timeslots. Not related to your PR but maybe a good thing to change since it causes duplicate requests.